client/src/styles/PortfolioTable.scss

```diff
@@ -76,16 +76,6 @@ $table-border-dark: 1px solid $dark;
         font-weight: 400;
       }
 
-      .arrow-icon {
-        cursor: pointer;
```

Comment:
I think we should keep the `cursor: pointer;` for the arrows to it's clear you can click them

Reply:
oh and probably the `margin-left: 0.3125rem;` (5px) as well to keep it the same. 
![image](https://github.com/JustFixNYC/who-owns-what/assets/16906516/c4d62abf-4096-4d68-9350-6199fe04b275)

Reply:
oh good call, I don't think I realized these were the icons that the classname was referring to